### PR TITLE
Enable Prow for Mixer

### PR DESCRIPTION
**Pull request by sebastienvas**
```release-note
NONE
```

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -116,6 +117,88 @@ presubmits:
...
+          secretName: mixer-codecov-token
+      - name: cache-ssd
+        hostPath:
+          path: /mnt/disks/ssd0
+    run_after_success:
+    - name: mixer-e2e-smoketest
+      context: prow/mixer-e2e-smoketest.sh
+      always_run: true
+      rerun_command: "@istio-testing smoke test this"
+      trigger: "((?m)^@istio-testing (smoke )?test this,?(\\s+|$)|(?m)^/test( all| bazel),?(\\s+|$))"
```
Comment (nclando) [inline prow/config.yaml]: can make this `/test( all| smoke)` please...also thinking we can move away from `@istio-testing smoke test` style comments in favor of promoting the `/test smoke` style  comments and maybe eventually require the `@istio-testing` ones

**Review by nlandolfi — CHANGES_REQUESTED** [review]
one minor thing

**Comment by nlandolfi** [comment]
also we should skip_reports until the job files go to master

File: prow/config.yaml (hunk 1 above)
Comment (sebastienvas) [reply prow/config.yaml]: Can we change all of those in another PR ?

**Comment by sebastienvas** [comment]
PTAL

**Review by nlandolfi — APPROVED** [review]
yeah just skip reports for e2e as well as discussed

File: prow/config.yaml (hunk 1 above)
Comment (nclando) [reply prow/config.yaml]: ya

**Review by nlandolfi — APPROVED** [review]

**Pull request merged**